src/App.tsx

```diff
@@ -1,112 +1,39 @@
-import { gql, useQuery } from "@apollo/client";
-import { useState } from "react";
-import { SearchBar, YuzuHeader, MarketHeader, WatchList } from "./components";
-import { WatchListQuery, usEquities, crypto, forex } from "./queries";
-
-import useStream from "./useStream";
-
-import djs from "dayjs";
-
-const today = djs().format("YYYY-MM-DD");
-
-// const buttons = ["U.S. Equities", "Crypto", "Forex"];
-
-const TopSymbols = {
-  "U.S. Equities": {
-    query: usEquities,
-    symbols: ["SPY", "VTI", "SOND"],
-    aggregates: { limit: 1, before: `${today}` },
-  },
-  Crypto: {
-    query: crypto,
-    symbols: ["BTC-USD", "ETH-USD", "SOL-USD", "DOGE-USD"],
-    aggregates: { limit: 1, before: `${today}` },
-  },
-  Forex: {
-    query: forex,
-    symbols: ["EUR-USD", "GBP-USD", "JPY-USD", "RUB-USD"],
-    aggregates: { limit: 2, before: `${today}` },
-  },
-};
-
-const initialWatchList = ["S:VTI", "S:SPY", "C:BTC-USD"];
+import { Routes, Route, Link, useParams, Outlet } from "react-router-dom";
+import { YuzuHeader, YuzuHome } from "./components";
 
 function App() {
-  const [selectedButton, setSelectedButton] = useState(
-    Object.keys(TopSymbols)[0]
+  return (
+    <Routes>
+      <Route path="/" element={<Layout />}>
+        <Route index element={<YuzuHome />} />
```

Comment:
Made the change to add `index` to this main route, which seems to have done the trick